src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -48,32 +46,19 @@ public double getCenteringPower() {
         }
     }
 
-    /** Intake motor is a Vortex*/
+    // Intake motor is a Vortex
     private final CANSparkFlex motor = new CANSparkFlex(IntakeConstants.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
 
-    // change the motor from neo550 to whatever it actually is
+    // Change the motor from neo550 to whatever it actually is
     private static final DCMotor dcMotor = DCMotor.getNeoVortex(1);
 
-    /** Centering motor is a NEO */
+    // Centering motor is a NEO 
```

Comment:
You don't have to change this to a normal comment. If you hover over a variable anywhere in the code that has a javadoc comment, it will show you the description in the comment. However, normal comments also work, so don't change it back.